test/confit-test.js

```diff
@@ -26,6 +26,10 @@ test('confit', function (t) {
 
 
     t.test('get', function (t) {
+        //setting process.env.env to development, should not change 'env:env'.
+        //This should be ignored and 'env:env' should solely depend on process.env.NODE_ENV
+        process.env.env = 'development';
```

Comment:
the test isn't actually testing the value of the string `config.get('env:env')`. Also should you set `process.env.env` to an arbitrary value?